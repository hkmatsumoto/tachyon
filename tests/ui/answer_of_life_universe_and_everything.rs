fn answer_to_life_universe_and_everything() -> i32 {
    42
}
