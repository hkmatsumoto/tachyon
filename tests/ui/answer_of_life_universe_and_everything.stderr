; ModuleID = 'top'
source_filename = "top"

define i32 @answer_to_life_universe_and_everything() {
entry:
  %_0 = alloca i32, align 4
  br label %bb0

bb0:                                              ; preds = %entry
  store i32 42, i32* %_0, align 4
  %0 = load i32, i32* %_0, align 4
  ret i32 %0
}
